invenio_app_rdm/theme/templates/invenio_app_rdm/search/search.html

```diff
@@ -0,0 +1,36 @@
+{#
+    Copyright (C) 2019 CERN.
+    Copyright (C) 2019 Northwestern University.
+
+    Invenio App RDM is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+#}
+{%- extends "invenio_theme/frontpage.html" %}
```

Comment:
Shouldn't we inherit from the `config.BASE_TEMPLATE` ?

Reply:
This is open to discussion! I did that because is seemed the easiest to so I can have the same style as the frontpage. But in general part of the discussion today would be to see the template architecture and from where to inherit or which blocks should be defined so they can be overriden. Or even which templates should come through config so in my instance I can pass my own. Currently I just exposed the whole search page template plus the header :)

Reply:
... yeah, my vote is still to inherit from `config.BASE_TEMPLATE` ;)  (frontpage is just a specialized page)